source/libs/cache.ts

```diff
@@ -46,7 +46,7 @@ export function set<TValue>(key: string, value: TValue, expiration?: number /* i
 }
 
 /* Accept messages in background page */
-if (!browser.runtime.getBackground) {
+if (!browser.runtime.getBackgroundPage) {
```

Comment:
The types update to `@types/firefox-webext-browser` found this type error. I wonder if this is a bug you never found as `getBackground` would have always been undefined.

Reply:
Oh wow, good catch. I think I need to update all of my webext modules 😬